docs/peggy/Peggy-Ceth-Account-Set.md

```diff
@@ -9,7 +9,9 @@ The transaction is priviledged, and only the admin account can set it.
 The admin account is the same account that updates the white list validators. The command is as follows:
 
 1. command format
-sifnodecli tx ethbridge update_ceth_receiver_account [cosmos-sender] [ceth-receiver-account]
+```
+sifnodecli tx ethbridge update_ceth_receiver_account $clp_admin_address $ceth-receiver-account
```

Comment:
@ElliotFriedman is this correct? should the sender account be the $clp_admin_address or $ethbridge-module address?

@ElliotFriedman do we need -from=$clp_admin_moniker or --from=$clp_admin_address too or smth?

Reply:
For the update_ceth_receiver_account, the first param is the sender, the second param is the receiver account.

Reply:
@musnit If you are going to merge this to master, you will also need to open up hotfix PR's to the testnet and develop branch.

Reply:
I would say we don't need to change the readme at all as it does a pretty good job of explaining what different variables are here.

Reply:
@ElliotFriedman  ideally all code in our readme can be copy-pasted and used directly, rather than needing manual modification, so these changes are intended to fix that

Reply:
@ElliotFriedman yeh which branch should this be based on?

Reply:
> For the update_ceth_receiver_account, the first param is the sender, the second param is the receiver account.

still unclear - so the first command takes the sender as a param directly, but the second command takes it as a flag? can you update the PR to confirm?

Reply:
We should be merging to develop. Then if we want to hotfix, we can merge up to testnet and master branches.

Reply:
@musnit Yes the sender is the admin, so that looks correct.